Tryouts/Plugins/ApplicationPlugins/MorganStanley.ComposeUI.ProcessExplorer/dotnet/src/LocalCollector/ProcessInfoCollector.cs

```diff
@@ -11,97 +11,80 @@
 // and limitations under the License.
 
 using System.Diagnostics;
-using LocalCollector.Communicator;
 using LocalCollector.Connections;
 using LocalCollector.EnvironmentVariables;
 using LocalCollector.Logging;
 using LocalCollector.Modules;
 using LocalCollector.Registrations;
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
+using ProcessExplorer.Abstractions.Entities;
+using ProcessExplorer.Abstractions.Entities.Connections;
+using ProcessExplorer.Abstractions.Infrastructure;
 
 namespace LocalCollector;
 
 public class ProcessInfoCollector : IProcessInfoCollector
 {
-    public ProcessInfoCollectorData Data { get; } = new();
+    public ProcessInfoCollectorData ProcessInformation { get; } = new();
```

Comment:
As you are using this inside lock blocks, it should be private, otherwise external code can cause concurrency issues